api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/PreviousNewValuePair.kt

```diff
@@ -0,0 +1,11 @@
+package org.pmiops.workbench.actionaudit.targetproperties
+
+data class PreviousNewValuePair(var previousValue: String?, var newValue: String?) {
+
+    val valueChanged: Boolean
+        get() {
+            return previousValue == null && newValue != null ||
+                    newValue == null && previousValue != null ||
+                    previousValue != newValue
```

Comment:
Does Kotlin implicitly convert `!=` into `!foo.equals(bar)`? If it doesn't, then this should probably use `.equals` over `!=`